src/code/views/image-browser-view.coffee

```diff
@@ -125,9 +128,9 @@ ImageSearch = React.createFactory React.createClass
             if @state.internalResults.length is 0 and (@state.searching or @state.externalResults.length > 0)
```

Comment:
This part set of conditional blocks (L128 - L133) is hard for me to reason about, because the intent isn't obvious. Any thoughts on how to make it more obvious?